### ISSUE TITLE
fails with multiple targetPlatforms on 1.1.3

### ISSUE THREAD

**Issue by shollander** [beryx/badass-runtime-plugin#6]
When trying to build for multiple platforms, e.g.
```
runtime {
    ...
    targetPlatform('linux-x64', '/usr/lib/jvm/jdk_x64_linux_hotspot_11_28')
    targetPlatform('win-x64', '/srv/jvm/win_x64_11.0.1')
    ...
 }
```

The build fails with the following error (seen when using `--stacktrace`):

```
...
Caused by: java.lang.UnsupportedOperationException
    at com.google.collect.ImmutableMap.put(ImmutableMap.java:450)
    at org.beryx.runtime.data.RuntimePluginExtension.targetPlatform(RuntimePluginExtension.groovy:74)
    at org.beryx.runtime.data.RuntimePluginExtension.targetPlatform(RuntimePluginExtension.groovy:73)
...
```
Note that this error occurs even with only a single `targetPlatform` specified.

This just started happening with version `1.1.3` of `org.beryx.runtime`. Seems to have been caused by the fix for #4.

**Comment by siordache** [comment]
Thanks for reporting. Fixed in 1.1.5.

**Comment by shollander** [comment]
Thanks for the quick fix! And thank you for this awesome library!
